<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>481</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    border: 1px solid black;
    border-radius: 5px;
}
QFrame {
    background-color: #121636;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <widget class="QLineEdit" name="nameEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>179</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #121636t;
    border-width: 1px;
    border-style: solid;
    border-color: white;
    border-radius: 10px;
    padding: 5px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>输入场景名称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="networkCountEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>179</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #121636t;
    border-width: 1px;
    border-style: solid;
    border-color: white;
    border-radius: 10px;
    padding: 5px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>输入网络数</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_log">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>179</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: blue;
    color: white;
    font-weight: bold;
    border-width: 1px;
    border-style: solid;
    border-color: gray;
    border-radius: 10px;
    padding: 5px;
}
QPushButton:hover {
    background-color: darkblue;
    border-color: blue;
}
QPushButton:pressed {
    background-color: lightblue;
}</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;创建网络规划&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
